teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(age, name, parent_permission)
```

Comment:
- [ ] Kindly use the colon notation to assign the `parent_permission` parameter to avoid ArgumentError as you already use the colon notation in the parent class(Person), so it will be like:
```ruby
super(age, name, parent_permission: parent_permission)
```